server/persistence/mysql/mysql.go

```diff
@@ -43,6 +44,10 @@ func NewFactory(ctx context.Context, config Config) (persistence.DBFactory, erro
 		for _, createStmt := range allCreateStmts {
 			_, err := db.ExecContext(ctx, createStmt)
 			if err != nil {
+				if config.CreateErrorIsOk {
+					log.Printf("error running CREATE: %q = %v", createStmt, err)
+					continue
+				}
```

Comment:
Why would this ever be ok?

Reply:
I was trying to fight this:
1. `docker-compose up cribbage-server` fails, but the sql container was stood up
2. fix the busted code, then re-up. now the sql tables exist, so it's ok if the creation fails.

Reply:
🤷‍♂️ 👍